index.js

```diff
@@ -156,3 +156,32 @@ cardBtn.forEach((btn, btnIndex) => {
     });
   });
 });
+
+function validateEmail() {
+  const emailInput = document.getElementById('email');
+  const email = emailInput.value.trim();
+  const errorMessage = document.getElementById('email-error');
+
+  if (email === '') {
+    errorMessage.textContent = 'Email is required';
+    errorMessage.style.display = 'block';
+    return false;
+  } if (email !== email.toLowerCase()) {
+    errorMessage.textContent = 'Email must contain only lowercase letters';
+    errorMessage.style.display = 'block';
+    return false;
+  }
+  const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+  if (!emailRegex.test(email)) {
+    errorMessage.textContent = 'Invalid email address';
+    errorMessage.style.display = 'block';
+    return false;
+  }
+  errorMessage.textContent = '';
+  errorMessage.style.display = 'none';
+  return true;
+}
+
+if (validateEmail()) {
+  document.getElementById('contact-form').submit();
+}
```

Comment:
- [ ] Ouch! there seems to be a bit of an issue here as the `validateEmail()` function is only called when the script loads according to your implementation but as per the project requirements, it should instead be called when the `submit` button is clicked. 
    - So first you'll need to attach the "click" event listener to the submit button and then pass a reference to the validate email function so it gets called whenever the button is clicked. 😉
- Here's the project requirement as a reminder:
  ![image](https://github.com/mumbereh/my-portfolio/assets/104968236/fb2493c9-d876-42eb-a023-26595cd0ca79)